keyboards/handwired/qc60/config.h

```diff
@@ -0,0 +1,46 @@
+/*
+Copyright 2012 Jun Wako <wakojun@gmail.com>
+Copyright 2015 Jack Humbert
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#ifndef CONFIG_H
```

Comment:
Could you remove the boilerplate and replace it with `#pragma once` ? 

eg, remove this: 
```h
#ifndef CONFIG_H
#define CONFIG_H
```
and replace it with `#pragma once`, and  also remove the `#endif` at the bottom of the file.